16 Lazy Loading/src/index.tsx

```diff
@@ -2,19 +2,41 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { Router, Route, IndexRoute, hashHistory } from 'react-router';
 import App from './components/app.tsx';
-import AboutPage from './components/about/aboutPage';
-import MembersPage from './components/members/membersPage';
-import MemberPage from './components/member/memberPage';
 
+//Loading single component in one chunk
+const lazyLoadAboutComponent = () => {
+  return {
+      getComponent: (location, callback)=> {
+        require.ensure([], require => {
+          callback(null, require('./components/about/aboutPage')["default"]);
+        }, 'AboutPage');
+      }
+    }
+};
+
+//Loading group of components in one chunk
```

Comment:
Cool, you can add a comment indicating the second one, just to let know other people browsing this sample that there are two possible approaches.